docs/content/interacting/transactional-writes.mdx

```diff
@@ -50,6 +50,17 @@ You have another type called `user` that can have a `follower` and `followed_by`
             this: {},
           },
         },
+        metadata: {
+          relations: {
+            owner: {
+              directly_related_user_types: [
+                { type: 'user' },
+                { type: 'user', relation: 'follower' },
+                { type: 'user', relation: 'followed_by' },
```

Comment:
I think we only need the follower here, not followed by

Reply:
```suggestion
```